### PR TITLE
Added ommited words in comment

### PR DESCRIPTION
**Pull request by SoumitraAgarwal**
"This will error" is grammatically incorrect, so I changed it to "This will give and error". "This will return an error is also fine"

### CODE REVIEW

File: auto_tests/messenger_test.c [hunk 1]
```diff
@@ -135,7 +135,7 @@ START_TEST(test_m_addfriend)
     if(m_addfriend(m, (uint8_t *)friend_id, (uint8_t *)good_data, really_bad_len) != FAERR_TOOLONG)
         ck_abort_msg("m_addfriend did NOT catch the following length: %d\n", really_bad_len);
 */
-/* this will error if the original m_addfriend_norequest() failed */
+/* this will give an error if the original m_addfriend_norequest() failed */
```
Comment (cebe) [inline auto_tests/messenger_test.c]: maybe just "This will fail" instead of "This will error"?

**Comment by ProMcTagonist** [comment]
http://dictionary.reference.com/browse/error

**Comment by GrayHatter** [comment]
For API documentation, you should use the word `return` not give

**Comment by SoumitraAgarwal** [comment]
I changed it to return now.

**Pull request merged**